### ISSUE TITLE
Problem with Oraxen 

### ISSUE THREAD

**Issue by Flox2000** [DerFrZocker/Custom-Ore-Generator#22]
I follow all the steps of the wiki but the plugin dosn't work with oraxen, I'm in 1.20.1 with the latest version of the plugin I get this error in console

> [01:46:14] [Server thread/ERROR]: Error occurred while enabling custom-ore-generator vdev-62-SNAPSHOT (Is it up to date?)
> 229 | java.lang.RuntimeException: Error while load data from file: plugins/custom-ore-generator/data/ore-config/Ruby.yml
> 230 | at de.derfrzocker.custom.ore.generator.utils.Config.(Config.java:49) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 231 | at de.derfrzocker.custom.ore.generator.impl.dao.LazyOreConfigCache.getOreConfig(LazyOreConfigCache.java:109) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 232 | at de.derfrzocker.custom.ore.generator.impl.dao.OreConfigYamlDao.lambda$getAll$0(OreConfigYamlDao.java:107) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 233 | at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
> 234 | at de.derfrzocker.custom.ore.generator.impl.dao.OreConfigYamlDao.getAll(OreConfigYamlDao.java:107) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 235 | at de.derfrzocker.custom.ore.generator.impl.CustomOreGeneratorServiceImpl.getOreConfigs(CustomOreGeneratorServiceImpl.java:239) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 236 | at de.derfrzocker.custom.ore.generator.CustomOreGenerator.onEnable(CustomOreGenerator.java:207) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 237 | at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> 238 | at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[purpur-1.20.1.jar:git-Purpur-2004]
> 239 | at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[purpur-1.20.1.jar:git-Purpur-2004]
> 240 | at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> 241 | at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:650) ~[purpur-1.20.1.jar:git-Purpur-2004]
> 242 | at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:561) ~[purpur-1.20.1.jar:git-Purpur-2004]
> 243 | at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:311) ~[purpur-1.20.1.jar:git-Purpur-2004]
> 244 | at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1117) ~[purpur-1.20.1.jar:git-Purpur-2004]
> 245 | at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323) ~[purpur-1.20.1.jar:git-Purpur-2004]
> 246 | at java.lang.Thread.run(Thread.java:833) ~[?:?]
> 247 | Caused by: org.bukkit.configuration.InvalidConfigurationException: Global tag is not allowed: tag:yaml.org,2002:io.th0rgal.oraxen.mechanics.provided.gameplay.noteblock.NoteBlockMechanic
> 248 | in 'reader', line 17, column 13:
> 249 | ORAXEN: !!io.th0rgal.oraxen.mechanics.pr ...
> 250 | ^
> 251 |  
> 252 | at org.bukkit.configuration.file.YamlConfiguration.loadFromString(YamlConfiguration.java:111) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> 253 | at org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:160) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> 254 | at de.derfrzocker.custom.ore.generator.utils.Config.load(Config.java:89) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 255 | at de.derfrzocker.custom.ore.generator.utils.Config.(Config.java:47) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 256 | ... 16 more
> 257 | Caused by: org.yaml.snakeyaml.composer.ComposerException: Global tag is not allowed: tag:yaml.org,2002:io.th0rgal.oraxen.mechanics.provided.gameplay.noteblock.NoteBlockMechanic
> 258 | in 'reader', line 17, column 13:
> 259 | ORAXEN: !!io.th0rgal.oraxen.mechanics.pr ...
> 260 | ^
> 261 |  
> 262 | at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:304) ~[snakeyaml-2.0.jar:?]
> 263 | at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> 264 | at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.0.jar:?]
> 265 | at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.0.jar:?]
> 266 | at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.0.jar:?]
> 267 | at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> 268 | at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.0.jar:?]
> 269 | at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.0.jar:?]
> 270 | at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.0.jar:?]
> 271 | at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> 272 | at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.0.jar:?]
> 273 | at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.0.jar:?]
> 274 | at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.0.jar:?]
> 275 | at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> 276 | at org.yaml.snakeyaml.composer.Composer.getNode(Composer.java:131) ~[snakeyaml-2.0.jar:?]
> 277 | at org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:157) ~[snakeyaml-2.0.jar:?]
> 278 | at org.yaml.snakeyaml.Yaml.compose(Yaml.java:575) ~[snakeyaml-2.0.jar:?]
> 279 | at org.bukkit.configuration.file.YamlConfiguration.loadFromString(YamlConfiguration.java:104) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> 280 | at org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:160) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> 281 | at de.derfrzocker.custom.ore.generator.utils.Config.load(Config.java:89) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 282 | at de.derfrzocker.custom.ore.generator.utils.Config.(Config.java:47) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> 283 | ... 16 more
> 
> [01:46:14] [Server thread/ERROR]: Error occurred while enabling custom-ore-generator vdev-62-SNAPSHOT (Is it up to date?) java.lang.RuntimeException: Error while load data from file: plugins/custom-ore-generator/data/ore-config/Ruby.yml at de.derfrzocker.custom.ore.generator.utils.Config.(Config.java:49) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?] at de.derfrzocker.custom.ore.generator.impl.dao.LazyOreConfigCache.getOreConfig(LazyOreConfigCache.java:109) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?] at de.derfrzocker.custom.ore.generator.impl.dao.OreConfigYamlDao.lambda$getAll$0(OreConfigYamlDao.java:107) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?] at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?] at de.derfrzocker.custom.ore.generator.impl.dao.OreConfigYamlDao.getAll(OreConfigYamlDao.java:107) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?] at de.derfrzocker.custom.ore.generator.impl.CustomOreGeneratorServiceImpl.getOreConfigs(CustomOreGeneratorServiceImpl.java:[239](https://mclo.gs/Cr4Dgto#L239)) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?] at de.derfrzocker.custom.ore.generator.CustomOreGenerator.onEnable(CustomOreGenerator.java:207) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?] at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:[281](https://mclo.gs/Cr4Dgto#L281)) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?] at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[purpur-1.20.1.jar:git-Purpur-2004] at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[purpur-1.20.1.jar:git-Purpur-2004] at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?] at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:650) ~[purpur-1.20.1.jar:git-Purpur-2004] at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:561) ~[purpur-1.20.1.jar:git-Purpur-2004] at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:311) ~[purpur-1.20.1.jar:git-Purpur-2004] at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1117) ~[purpur-1.20.1.jar:git-Purpur-2004] at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323) ~[purpur-1.20.1.jar:git-Purpur-2004] at java.lang.Thread.run(Thread.java:833) ~[?:?] Caused by: org.bukkit.configuration.InvalidConfigurationException: Global tag is not allowed: tag:yaml.org,2002:io.th0rgal.oraxen.mechanics.provided.gameplay.noteblock.NoteBlockMechanic in 'reader', line 17, column 13: ORAXEN: !!io.th0rgal.oraxen.mechanics.pr ... ^
> at org.bukkit.configuration.file.YamlConfiguration.loadFromString(YamlConfiguration.java:111) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> at org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:160) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> at de.derfrzocker.custom.ore.generator.utils.Config.load(Config.java:89) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> at de.derfrzocker.custom.ore.generator.utils.Config.(Config.java:47) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> ... 16 more
> Caused by: org.yaml.snakeyaml.composer.ComposerException: Global tag is not allowed: tag:yaml.org,2002:io.th0rgal.oraxen.mechanics.provided.gameplay.noteblock.NoteBlockMechanic
> in 'reader', line 17, column 13:
> ORAXEN: !!io.th0rgal.oraxen.mechanics.pr ...
> ^
> 
> at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:304) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:369) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:348) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:323) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:209) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.getNode(Composer.java:131) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:157) ~[snakeyaml-2.0.jar:?]
> at org.yaml.snakeyaml.Yaml.compose(Yaml.java:575) ~[snakeyaml-2.0.jar:?]
> at org.bukkit.configuration.file.YamlConfiguration.loadFromString(YamlConfiguration.java:104) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> at org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:160) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
> at de.derfrzocker.custom.ore.generator.utils.Config.load(Config.java:89) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> at de.derfrzocker.custom.ore.generator.utils.Config.(Config.java:47) ~[custom-ore-generator-dev-62-SNAPSHOT.jar:?]
> ... 16 more

**Comment by NetsukiiDev** [comment]
Me too, here's my config:

```
# Only edit this file if you 10000% sure you know what you are doing. 
# You can set everything via the plugin commands, no need to edit this file manully.

value:
  ==: CustomOreGenerator#OreConfig
  name: Orax
  material: NOTE_BLOCK
  ore-generator: VANILLA_MINABLE_GENERATOR
  block-selector: COUNT_RANGE
  activated: true
  generated-all: true
  replace-materials:
  - STONE
  - DEEPSLATE
  custom-data:
    ORAXEN: !!io.th0rgal.oraxen.mechanics.provided.gameplay.noteblock.NoteBlockMechanic {}
  ore-generator-settings:
    ==: CustomOreGenerator#OreSettingsContainer
    ore-settings:
      VEIN_SIZE: 40.0
  block-selector-settings:
    ==: CustomOreGenerator#OreSettingsContainer
    ore-settings:
      MINIMUM_HEIGHT: -30.0
      VEINS_PER_CHUNK: 50.0
```

**Comment by editon96** [comment]
I read that that Plugin supports only to 1.19.4 so 1.20.1 are curently not suported...

**Comment by NetsukiiDev** [comment]
dev version actually support 1.20.1

**Comment by DerFrZocker** [comment]
Fixed in version 2023.07.9

You need to reconfigured your ores.
